Dockerfile

```diff
@@ -3,13 +3,26 @@ ENV VAULT_VERSION 1.8.4
 
 
 WORKDIR /vault
-RUN apk add --no-cache git openssh gcc musl-dev curl gnupg unzip
-
+RUN apk add --no-cache git openssh gcc musl-dev curl gnupg unzip go make bash
```

Comment:
Maybe just change the builder image to `golang:latest` which will include a Go environment ready to use